### PR TITLE
builtin/aws/alb: clamp alb name per aws limits

### PR DESCRIPTION
**Pull request by krantzinator**

### CODE REVIEW

File: builtin/aws/alb/releaser.go [hunk 1]
```diff
@@ -51,6 +51,12 @@ func (r *Releaser) Release(
 		lbName = "waypoint-" + src.App
 	}
 
+	// We have to clamp at a length of 32 because the Name field to 
+	// CreateLoadBalancer requires that the name is 32 characters or less.
+	if len(lbName) > 32 {
+		lbName = lbName[:32]
```
Comment (briancain) [inline builtin/aws/alb/releaser.go:57]: Maybe `log.Debug` here that we had to clamp the LB name due to the character restriction, with the original one in the message?

File: .changelog/1225 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```
Comment (briancain) [inline .changelog/1225:1]: Should be `1225.txt`

Comment (krantzinator) [reply .changelog/1225:1]: ah thanks

File: builtin/aws/alb/releaser.go (hunk 1 above)
Comment (krantzinator) [reply builtin/aws/alb/releaser.go:57]: Good idea! We do this clamping in several places -- should I add debugs to all of those in separate PRs or just in this one you think?

Comment (briancain) [reply builtin/aws/alb/releaser.go:57]: Hrm, I don't mind it in here if they're all grouped into the same commit! I think that makes sense.

**Review by briancain — APPROVED** [review]
👍🏻 looks good to me!

**Pull request merged**